tests/test_misc.py

```diff
@@ -57,24 +57,24 @@ def test_dedup():
 
 
 def test_run():
-    out, err = vistir.misc.run(["python", "-c", "print('hello')"], nospin=True)
+    out, err = vistir.misc.run([sys.executable, "-c", "print('hello')"], nospin=True)
     assert out == "hello"
-    out, err = vistir.misc.run(["python", "-c", "import ajwfoiejaoiwj"], nospin=True)
+    out, err = vistir.misc.run([sys.executable, "-c", "import ajwfoiejaoiwj"], nospin=True)
```

Comment:
```suggestion
    out, err = vistir.misc.run([r"{0}".format(sys.executable), "-c", "import ajwfoiejaoiwj"], nospin=True)
```

Reply:
I don't understand why this differs.

```pycon
>>> a = 'C:\\what\\not'
>>> a
'C:\\what\\not'
>>> r'{0}'.format(a)
'C:\\what\\not'
>>> a = 'C:\what\not'
>>> a
'C:\\what\not'
>>> r'{0}'.format(a)
'C:\\what\not'
```

Reply:
@hroncok that initially confused me as well, but the difference is that `\n` is a valid character (newline), so python will not make any effort to escape it during `repr` calls.  For example:

```python
>>> a = 'C:\what\not'
>>> a
'C:\\what\not'
>>> print(a)
C:\what
ot
```

The goal of using raw strings is a bit disingenuous here and a bit of a shortcut, the "real" fix would be to just posix-ify these paths with `pathlib.Path().as_posix()` calls like we do everywhere else (with an import exception which falls back to `pathlib2` for python 2 if we want to avoid just using `vistir.compat.Path`).

Reply:
My point is that formating something into a raw string literal clearly doesn't do anything. AFAIK r'' is the same thing as ''.

Reply:
hm so it seems, was not paying attention to the format call super closely.  We need to swap this out then.